core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -76,9 +74,14 @@
         border-bottom: 2px solid $color-palette-secondary-500;
     }
 
+    .p-tabview .p-tabview-nav li:first-child .p-tabview-nav-link {
+        border: none !important;
```

Comment:
We try to avoid `!important`, is not any other way?

Reply:
Yep, i removed it on the new commit